app/services/make_an_offer.rb

```diff
@@ -0,0 +1,15 @@
+class MakeAnOffer
+  Response = Struct.new(:successful?, :application_choice)
+
+  def initialize(application_choice:, offer_conditions:)
+    @application_choice = application_choice
+    @offer_conditions = offer_conditions
+  end
+
+  def call
+    @application_choice.status = @offer_conditions.present? ? :conditional_offer : :unconditional_offer
```

Comment:
Feels like this would be better as a single conditional (`if @offer_conditions`), what do you think?

Reply:
:+1: